jupyterlab_gcedetails/src/components/machine_type_select.tsx

```diff
@@ -270,8 +271,12 @@ export class NestedSelect extends React.Component<
 
     return (
       <div className={STYLES.nestedSelect}>
-        {label && <label>{label}</label>}
+        {/* {label && <label>{label}</label>} */}
```

Comment:
Can you remove this rather than leave it commented if it's not used?

Reply:
Ah that's my bad, I forgot to remove it - good catch!